client/src/components/game/gui/panels/map/MapPanel.jsx

```diff
@@ -14,7 +70,26 @@ function MapPanel({ onCloseCallback }) {
               icon={mapIcon}
               onCloseCallback={onCloseCallback}
             >
-                <div className="inner-cont" />
+                <Map
+                  center={leafletConfig.center}
+                  zoom={leafletConfig.zoom}
+                  maxZoom={4}
+                  minZoom={1}
+                  crs={CRS.Simple}
+                  doubleClickZoom={false}
+                >
+                    <TileLayer onClick={(e) => console.log("Hello")} url="http://127.0.0.1:4567/map/{z}/{x}/{y}.png" />
```

Comment:
```url="http://127.0.0.1:4567/map/{z}/{x}/{y}.png"```
This will need to change to connect to where ever the server is.
I just updated master to expose the server URL that it uses for the websocket connection, but for HTTP too. It will use the correct host address and protocol (HTTP for dev, HTTPS for prod) as I think the deployment server only accepts HTTPS.
Should just need to import the ApplicationState and use this:
```url=`${ApplicationState.httpServerURL}/map/{z}/{x}/{y}.png` ```

Reply:
Oh thanks, forgot to ask about this one.